docs/roadmap.md

```diff
@@ -0,0 +1,124 @@
+collaboration.
+
+In addition to end users presenting to the SIG, [the CDF User Stories Program](https://cd.foundation/blog/2023/02/07/launching-cdf-user-stories-first-up-fidelity-investments/)
+features end user organizations to share their requirements, use cases, challenges, and the initiatives
+they have are in order to share them with the communities on an ongoing basis.
+
+#### Supply Chain Maturity
+
+Supply Chain Maturity metrics are supplemental to the many details [Dora](https://www.devops-research.com/research.html)
+elides and are helpful in identifying gaps in the supply chain that will improve Dora metrics.
+Supply Chain Maturity metrics provide milestones like automated rollout, automated rollback, and deployment
+tracking that provide the tooling and insights needed to minimize recovery time.
+
+The initial version of the [Supply Chain Maturity metrics](https://github.com/cdfoundation/sig-software-supply-chain/blob/main/docs/supply-chain-maturity.md)
+has been developed by the SIG in a collaborative manner and contributions are expected and welcomed in
+order to improve it.
+
+### Next
+
+#### Establishing Shared Vocabulary
+
+Various groups within the CDF such as [SIG Interoperability](https://github.com/cdfoundation/sig-interoperability)
+and [SIG Events](https://github.com/cdfoundation/sig-events) have been working on
+developing a shared vocabulary to help enable the further developments in corresponding
+areas. The work is done within the SIG Interoperability repository and the community is
+encouraged to contriubute to corresponding documents listed below.
+
+* [SIG Interoperability Tools Vocabulary](https://github.com/cdfoundation/sig-interoperability/blob/main/docs/tools-terminology.md)
+* [SIG Interoperability Pipelines Vocabulary](https://github.com/cdfoundation/sig-interoperability/blob/main/docs/pipelines-terminology.md)
+
+The vocabulary documents currently lack terminology around Software Supply Chain and the SIG
+Software Supply Chain could potentially look into contributing to the shared vocabulary.
+
+#### Supply Chain Security
+
+Supply Chain Security is a critical aspect with regards to Continuous Delivery, from the
+software that goes through the pipelines (e.g., source code, artifacts) as well as the
+pipelines themselves.
+
+The SIG Software Supply Chain charter explicitly states that the SIG aims to contribute
+Software Supply Chain efforts and security is one of the key areas.
+
+### Later
+
+#### Events Standardizations
+
+CDF [CDEvents](https://cdevents.dev) project works on creating a common specification for
+Continuous Delivery events in order to achieve interoperability within the CD ecosystem.
+The current version of the specification does not have Software Supply Chain related
+events and the SIG Software Supply Chain could look for ways to contribute such events
+to the project.
+
+Please note that this work depends on the earlier work, Establishing Shared Vocabulary.
+
+
+## Contributors
```

Comment:
Is this intended to be "Contributors to this Document" or "Contributors to the SIG"? I assume the former, but it isn't clear -- suggest framing it that way for clarity.